texts/epn_paper.md

```diff
@@ -0,0 +1,72 @@
+
+To help the community members orient themselves in a new setting, we prepared basic guidelines for all online event participants: organizers, moderators, and the audience.
+Once we felt sufficiently confident, we ran an trial workshop with our direct colleagues as both speakers and the audience.
+We used this test run to try out different formats of accommodating interaction between the audience and the speaker, check out unexpected technical problems, and collect the impressions from a broader group of colleagues.
+More importantly, however, we observed that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.
+Some of our attendees planned their own "watching parties" with a number of participants viewing as a group that pre-discussed their questions and afterwards shared them with the speaker.
+Overall we received overwhelmingly positive feedback from all participants.
+
+With this experience, we were ready to hit the ground running and managed to organize a virtual session within only a few days after the APS March Meeting was cancelled.
+We are now organizing the biweekly VSF Long Range Colloquium series.
+We also actively welcome community initiatives for organizing virtual conferences using our platform, and are excited to already have the first of such conference applications in progress.
+Our experience with organizing online conferences and colloquia has shown that they provide a stimulating environment and foster fruitful discussions.
+
+<!-- #### We are confident that we will see an explosive growth of online events. -->
+
+With more and more people getting used to online meetings and seminars -- and with online conferencing software continually improving -- we expect and hope that online events keep growing. 
+Not because of our platform; but because they allow for broad, easy accessibility and because of their potential contribution to reducing carbon emission.
+Our effort towards online conferences was not the first and is certainly no longer one of few.
+Online colloquia and seminars are organized by many science departments and research institutes.
+Virtual workshops and conferences are also on the rise, although at a lower rate due to requiring more coordination and planning.
+We also expect to see more organizations like [researchseminars.org](https://researchseminars.org) that distribute announcements in order to help navigate the rapidly growing volume of online events.
+
+<!-- #### You should definitely get involved in the online events, and we are here to help. -->
+
+Online conferencing is emerging right now because there are no alternatives until the pandemic is over, lockdowns are lifted, and travel is restored.
+While our hands are forced now, we hope that the research community will grow accustomed to this long overdue change and will see the value of virtual conferences in their own right.
```

Comment:
```suggestion
While our hands are forced, we hope that the scientific community will grow accustomed to this long overdue change and will see the value of virtual conferences in their own right.
```
Removed 'now' because of previous sentence. Scientific community instead of research, broader?